app/pages/project/index.spec.js

```diff
@@ -49,45 +49,100 @@ const resources = {
   ]
 };
 
-const project = apiClient.type('projects').create({
-  id: 'a',
-  display_name: 'A test project',
-  experimental_tools: [],
-  get(type) {
-    return Promise.resolve(resources[type]);
-  },
-  links: {
-    active_workflows: ['1', '2', '3', '4', '5'],
-    avatar: { id: '1' },
-    background: { id: '1' },
-    owner: { id: '1' }
+function mockPanoptesResource(type, options) {
+  const resource = apiClient.type(type).create(options);
+  if (!resource.save.restore) {
```

Comment:
Not a blocker at all but revealing the intention here to stub if not stubbed already could maybe be commented / extracted to function? 

I initially though it would call the save and the code might be unintentionally modified to do so in future, e.g `if (!resource.save().restore)`. That said it's probably just me being paranoid

Reply:
A better approach would be to clean up the stubs in an `after` block. This check is really only here because the stubs in the email spec test interfere with the tests here, which isn't good.